src/utils/getMyVoteData.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file getMyVoteData.js
+ * @author zhouminghui
+ * = W = 注释我之后补一下，见谅。。。。
+ */
+import getWallet from './getWallet';
+import getConsensus from './getConsensus';
+import hexCharCodeToStr from './hexCharCodeToStr';
+import formatUtc from './formatUtc';
+import formatUtcToDate from './formatUtcToDate';
+
+export default function getMyVoteData(currentWallet, startIndex) {
+    let dataList = [];
+    if (!currentWallet) {
+        return dataList;
+    }
+    const wallet = getWallet(currentWallet.privateKey);
+    const consensus = getConsensus(wallet);
+    const ticketsInfo = JSON.parse(
+        hexCharCodeToStr(
+            consensus.GetPageableTicketsInfoToFriendlyString(
+                currentWallet.publicKey, startIndex.page, startIndex.pageSize
+            ).return
+        )
+    );
+    const ticketsInfoList = ticketsInfo.VotingRecords;
+    const VotingRecordsCount = ticketsInfo.VotingRecordsCount;
+    if (ticketsInfoList) {
```

Comment:
const ticketsInfoList = ticketsInfo.VotingRecords || [];
ticketsInfoList.map((item, index) => {});

Reply:
Okay, thank you. I've been taught.